crates/router/src/utils.rs

```diff
@@ -159,7 +160,13 @@ impl<E> ConnectorResponseExt
 pub fn get_payment_attempt_id(payment_id: impl std::fmt::Display, attempt_count: i16) -> String {
     format!("{payment_id}_{attempt_count}")
 }
-
+pub fn get_formatted_date_time(format: &str) -> CustomResult<String, errors::ConnectorError> {
+    let format = format_description::parse(format)
```

Comment:
We can use the [`format_description`](https://docs.rs/time/latest/time/macros/macro.format_description.html) macro instead.